src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,26 @@
 package core.basesyntax;
 
+import core.basesyntax.figure.Figure;
+import core.basesyntax.supplier.FigureSupplier;
+
 /**
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
+    private static int FIGURE_AMOUNT = 6;
+    private static int halfOfArray = (FIGURE_AMOUNT + FIGURE_AMOUNT % 2) / 2;
```

Comment:
No reason to move this to constant, because this value could be calculated, and this calculation is simple